lib/packages/core/lib/src/views/media/media_forwarding.dart

```diff
@@ -76,6 +91,59 @@ class _LMChatMediaForwardingScreenState
     );
   }
 
+  Widget _buildMediaPreview() {
+    return Column(
+      children: [
+        Expanded(
+          child: _buildMainPreview(),
+        ),
+        if (mediaList.length > 1) _buildPreviewBar(),
+      ],
+    );
+  }
+
+  Widget _buildMainPreview() {
```

Comment:
rename _buildMainPreview() to _buildMediaPreview, and _buildMediaPreview() to _buildPreview()

Reply:
Kyon bhai? Ainvayin? Mauj mein?

Reply:
if you think _buildMediaPreview() and _buildMainPreview() is better then keep it.